### ISSUE TITLE
Safari unable to load generated pages

### ISSUE THREAD

**Issue by erantapaa** [gatsbyjs/gatsby#264]
Perhaps this is expected, but I'll mention it anyway...

The `index.js` created by `gatsby new ...` doesn't load on Safari 8.0.5 (OS X 10.10.3).

To replicate:

    $ gatsby new blah
    $ cd blah
    $ gatsby develop

When loading `http://0.0.0.0:8000` Safari just shows a blank page and reports the following errors in the console:

    [Error] TypeError: undefined is not a function (evaluating 'Object.assign({}, this.props, {
          "style": styles
        })')
      perform (undefined, line 18)
      batchedUpdates
      batchedUpdates
      _renderNewRootComponent
      ReactMount__renderNewRootComponent
      _renderSubtreeIntoContainer
      render
      React_render
      (anonymous function)
      exports
      (anonymous function)
      loadConfig
      eval code
      eval ([native code], line 0)
      (anonymous function) (bundle.js, line 2902)
      __webpack_require__ (bundle.js, line 556)
      fn (bundle.js, line 87)
      eval code
      eval ([native code], line 0)
      (anonymous function) (bundle.js, line 586)
      __webpack_require__ (bundle.js, line 556)
      (anonymous function) (bundle.js, line 579)
      global code (bundle.js, line 580)
    [Error] Failed to load resource: unsupported URL (performance-now.map, line 0) webpack:///~/react-responsive-grid/~/react-page-width/~/raf/~/performance-now/lib/performance-now.map

**Comment by benstepp** [comment]
Safari 8.0 doesn't support Object.assign(), and babel uses it all over the place when transforming JSX. You're going to have to polyfill it. Here's one of the easier ways to do so.

```bash
npm install --save babel-plugin-transform-object-assign
```

```js
// .babelrc
{
  "presets": ['react', 'es2015', 'stage-0'],
  "plugins": [
    'transform-object-rest-spread',
    'transform-object-assign'
  ]
}
```

**Comment by KyleAMathews** [comment]
Hmmm... perhaps we should just add this plugin to the starters given there's a decent % of browsers in use that don't support Object.assign yet like Safari 8.

**Comment by KyleAMathews** [comment]
@benstepp actually could you just add this to core in #277 actually? Any reason not to?

**Comment by benstepp** [comment]
Sounds good.

**Comment by KyleAMathews** [comment]
Fixed in https://github.com/gatsbyjs/gatsby/pull/279 and released in 0.11
